ui/app/src/components/PreviewDropTargetsPanel/PreviewDropTargetsPanel.tsx

```diff
@@ -127,67 +156,117 @@ export function PreviewDropTargetsPanel() {
+function ContentTypeItem(props: ContentTypeItemContentProps) {
+  const { contentType, contentTypesBranch } = props;
+  const theme = useTheme();
+  const { backgroundColor, textColor } = getAvatarWithIconColors(
+    contentTypesBranch.byId[contentType.id].name,
+    theme,
+    darken
+  );
   return (
-    <>
-      <div className={classes.select}>
-        <Select
-          value={dropTargetsBranch.selectedContentType || ''}
-          displayEmpty
-          onChange={(event: any) => {
-            event.stopPropagation();
-            handleSelectChange(event.target.value);
-          }}
-        >
-          <MenuItem value="" disabled>
-            {formatMessage(translations.selectContentType)}
-          </MenuItem>
-          {contentTypes?.map((contentType: ContentType, i: number) => {
-            const { backgroundColor, textColor } = getAvatarWithIconColors(
-              contentTypesBranch.byId[contentType.id].name,
-              theme,
-              darken
-            );
-            return (
-              <MenuItem value={contentType.id} key={i}>
-                <Box display="flex" alignItems="center">
-                  <Box
-                    sx={{
-                      mr: 1,
-                      flexShrink: 0,
-                      width: '24px',
-                      height: '24px',
-                      borderRadius: '20px',
-                      overflow: 'hidden',
-                      backgroundColor,
-                      borderColor: textColor,
-                      borderStyle: 'solid',
-                      borderWidth: '1px'
-                    }}
-                  />
-                  <Box sx={{ flexShrink: 1, flexGrow: 1, textOverflow: 'ellipsis' }} title={contentType.name}>
-                    {contentType.name}
-                  </Box>
-                </Box>
-              </MenuItem>
-            );
-          })}
-        </Select>
-      </div>
-      <List>
-        <SuspenseWithEmptyState
-          resource={dropTargetsResource}
-          withEmptyStateProps={{
-            emptyStateProps: {
-              title: dropTargetsBranch.selectedContentType
-                ? formatMessage(translations.noResults)
-                : formatMessage(translations.chooseContentType)
-            }
-          }}
-        >
-          <DropTargetsList resource={dropTargetsResource} onSelectedDropZone={onSelectedDropZone} />
-        </SuspenseWithEmptyState>
-      </List>
-    </>
+    <Box display="flex" alignItems="center" overflow="hidden">
+      <Box
+        sx={{
+          mr: 1,
+          flexShrink: 0,
+          width: '24px',
+          height: '24px',
+          borderRadius: '20px',
+          overflow: 'hidden',
+          backgroundColor,
+          borderColor: textColor,
+          borderStyle: 'solid',
+          borderWidth: '1px'
+        }}
+      />
+      <Box sx={{ flexShrink: 1, flexGrow: 1, textOverflow: 'ellipsis', overflow: 'hidden' }} title={contentType.name}>
```

Comment:
This should be a typography and you get the noWrap for free.

Reply:
On second thought, this whole component should be a MenuItem or ListItemButton (based on a prop) and handle the differences in the two renderings. I reckon the type colour circle goes in both cases inside the ListItemIcon ([see example](https://mui.com/material-ui/react-menu/#icon-menu)).

The whole thing would be something like
```ts
const { contentType, type, onClick?, etc } = props
const Compoenent = type === 'list' ? ListItemButton : MenuItem
...
  <Component ...>
    <ListItemIcon>
      <Box ...
    <ListItemText>
      {contentType.name}
...
```

Reply:
Per conversation we'll leave the ListItemButton/Menu item out of the component to avoid splitting props and issues with Select component

Reply:
Component updated with ListItemIcon and ListItemText though